README.md

```diff
@@ -127,5 +127,22 @@ with context.new_entry_span(op=str('https://github.com/apache/skywalking')) as s
 
 Check [the FAQ page](docs/FAQ.md) or add the FAQs there.
 
+## For Developers
+
+### Steps to get an operational virtual environment:
+
+1. `git clone https://github.com/apache/skywalking-python.git`
+2. `cd skywalking-python/scripts` (*make sure you actually go into the directory since the scripts use relative paths*)
+3. Run the script for your relevant OS to create a virtual environment folder in the project root (*skywalking-python/venv*) and install all the necessary requirements
+
+**Make sure that when the `python` command is executed on your workstation, the binary it references is python 3.5 or newer!**
```

Comment:
IIRC this can be automatically determined right?

Reply:
I'll have to double-check but I think you can determine either python3 or perhaps python37 but IDK about specifically python35-python38. And sometimes python3 is not implemented as an option (on my PC, writing python3 in PowerShell redirects me to the Microsoft Store :D